cli/cluster.go

```diff
@@ -44,6 +44,8 @@ Commands:
             -d, --default             set as default cluster
             --git-url=<giturl>        git URL
             --no-git                  skip git configuration
+            --docker-url=<dockerurl>  Docker URL
```

Comment:
Hmm, I think that this may be confused for a Docker daemon URL, any ideas for disambiguation?